token/src/token.rs

```diff
@@ -0,0 +1,89 @@
+use bulletproofs::{BulletproofGens, PedersenGens};
+use curve25519_dalek::scalar::Scalar;
+use zkvm::*;
```

Comment:
let's be more explicit about what we import? The rationale is that we are still not sure what zkvm API looks like, so exact usage in `./token`, `./blockchain` etc would inform us. The `use zkvm::*` does not tell us which types are actually used.

Reply:
made the imports more explicit! I think it would be really great to break `zkvm` into some submodules to make the imports more wieldy (but maybe we can try to define those based on usage :) )

Reply:
totally!